### PR TITLE
Mark chroma_sample_position as deprecated

### PR DESCRIPTION
**Pull request by rosetta-jpn**
Signed-off-by: Hirokazu Honda <hiroh@chromium.org>

### CODE REVIEW

**Comment by rosetta-jpn** [comment]
@sreerenjb could you review this patch?

**Review by sreerenjb — COMMENTED** [review]
Please use the @deprecated tag for doxygen. Otherwise lgtm.

File: va/va_dec_av1.h [hunk 1]
```diff
@@ -255,6 +255,7 @@ typedef struct  _VADecPictureParameterBufferAV1
             uint32_t        color_range                                 : 1;
             uint32_t        subsampling_x                               : 1;
             uint32_t        subsampling_y                               : 1;
+            /** chroma_sampling_position is deprecated */
```
[the remark this answers is not in the document]
Comment (rosetta-jpn) [reply va/va_dec_av1.h]: Done. Thanks!

**Comment by rosetta-jpn** [comment]
gentle ping!

**Review by XinfengZhang — APPROVED** [review]

**Pull request merged**